app.py

```diff
@@ -1,2 +1,83 @@
-from flask import Flask, render_template, request, redirect, url_for
+from flask import Flask, render_template, request, redirect, url_for, abort
 from flask_sqlalchemy import SQLAlchemy
+import string
+import random
+
+app = Flask(__name__)
+
+app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///urls.db'
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+
+db = SQLAlchemy(app)
+
+@app.before_first_request
+def create_tables():
+    db.create_all()
+
+class Urls(db.Model):
+    id = db.Column("id_", db.Integer, primary_key=True)
+    long = db.Column("long", db.String())
+    short = db.Column("short", db.String(5))
+
+    def __init__(self, long, short):
+        self.long = long
+        self.short = short
+
+    def __str__(self):
+        return f'Long URL:{self.long}, short URL:{self.short}'
+
```

Comment:
could you move this to a models folder in your project struct? - obviously it's a small app, so not a big issue